scripts/add_metrics_to_readme.sh

```diff
@@ -9,6 +9,30 @@ function getMetrics() {
     done
 }
 
+function failureConfigChange() {
+	local logstashCID
+	logstashCID=$( docker ps -a | grep 'logstash-exporter-logstash-1' | awk '{print $1}' )
+	local logstashConf
+	logstashConf='/usr/share/logstash/pipeline/logstash.conf'
+
+	docker exec -it "$logstashCID" sh -c "echo 'Wrong Config' >> $logstashConf"
+
+	# reload logstash
+	# "logstash pid should be always 1 AFAIK" ~ Kuskoman, 2024
+	docker exec -it "$logstashCID" sh -c "kill -1 1"
```

Comment:
assign 1 to a variable, so it will be more readable

Reply:
Done 🫡